src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrapCheckTest.java

```diff
@@ -34,8 +34,7 @@
     extends BaseCheckTestSupport {
     @Override
     protected String getPath(String filename) throws IOException {
-        return super.getPath("checks" + File.separator
-                + "whitespace" + File.separator + filename);
+        return super.getPath("checks"+File.separator+"whitespace"+File.separator+"nolinewrap"+File.separator+filename);
```

Comment:
@divsidhu-26 
The line should be less than 100 chars.
https://travis-ci.org/checkstyle/checkstyle/jobs/209401534#L358